examples/using-javascript-transforms/src/static/fonts/fontawesome/style.css

```
@@ -1595,7 +1595,7 @@
   content: "\f16e";
 }
 
-.fa-adn:before {
+.fa-and:before {
```

```suggestion
.fa-adn:before {
```
this is not a typo here ;)

---

I used a tool to fix these. There are always limitations.